nym-data-observatory/README_SQLX.md

```diff
@@ -39,35 +39,25 @@ If it's outside of code (i.e. when running `cargo check`)
 - make sure password doesn't have any special characters that could be
   interpreted by the command line/shell weirdly, like `$#\` etc.
 
-## Cannot generate `sqlx-data.json`
+## Offline query data looks like this
 
-In order for `sqlx` to generate schema for "offline" work (without DB
-connection), as of `v0.6.3` you first **need an active DB connection**.
-
-So make sure
-
-- DB is running
-- `DATABASE_URL` is set correctly
-- `SQLX_OFFLINE` isn't exported to true
-
-Then run `cargo sqlx prepare`
-
-After you have the file, you can ignore `DATABASE_URL` and terminate the DB
-instance. This file represents the DB schema, so when your migrations change,
-you'll need to re-generate it
-
-Make sure to commit the file to VCS if you want to avoid re-doing this again on
-each machine (e.g. other developers, CI).
+```
+.sqlx/
+├─ new_file
+├─ query-249faa11b88b749f50342bb5c9cc41d20896db543eed74a6f320c041bcbb723d.json
+├─ query-aff7fbd06728004d2f2226d20c32f1482df00de2dc1d2b4debbb2e12553d997b.json
+├─ ...
+├─ query-e53f479f8cead3dc8aa1875e5d450ad69686cf6a109e37d6c3f0623c3e9f91d0.json
+```
 
-## Generated `sqlx-data.json` looks like this
+The offline mode for the queries uses a separate file per `query!()` invocation
 
-```json
-{
-  "db": "PostgreSQL"
-}
-```
+Each workspace member that works with `sqlx` has `.sqlx` directory, containing
+its own schema description. This allows compile-time checks without needing a
+live DB connection (so called `OFFLINE_MODE`).
 
-after running `cargo sqlx prepare`
+To initialize those files, you need to run `cargo sqlx prepare` with a live
```

Comment:
I think what might be useful in here is a comment to show how to create that live connection to DB and how to update the schema. Because so far everything was happening automagically behind the scenes in `build.rs` that was making the ephemeral db

Reply:
"live db" in this instance means running postgres and there is  a convenience script `pg_up.sh` for that in this module. For the rest it's as is written here
```
cargo sqlx prepare
```